### PR TITLE
Display name of user selected when delete btn is clicked

### PR DESCRIPTION
**Pull request by LouiaR**
@LouiaR  you have marge conflict try to fix it then push your changes, please.

### CODE REVIEW

File: frontend/src/Components/LandingPage/index.js [hunk 1]
```diff
@@ -27,3 +27,15 @@ const LandingPage = () => {
 };
 
 export default LandingPage;
+
+
+
+
+
+// function addSpaceBetweenCatName(categories, title) {
```
Comment (kkarimi) [inline frontend/src/Components/LandingPage/index.js]: @LouiaR please remove commented code

File: frontend/src/Components/Organisation/OrganisationCard.js [hunk 2]
```diff
@@ -4,22 +4,31 @@ import SingleOrganisation from './SingleOrganisation';
...
-        <h4>{Organisation}</h4>
-        <h6>
-          Area: {Area} | Borough: {org.Borough}
-        </h6>
-        <div className="health-advice-process">
-          <p> - {Services}</p>
-          <p> - {Process}</p>
-        </div>
+const OrganisationCard = ({
+  org,
```
Comment (kkarimi) [inline frontend/src/Components/Organisation/OrganisationCard.js]: what is the difference between `org` and `organisation` props? we should be explicit where we can with the names

File: frontend/src/helpers.js [hunk 3]
```diff
@@ -149,6 +149,16 @@ function categoryNameMaker(name) {
     : null;
 }
 
+function addSpace(cate, cateTitle) {
+ return cate.map(category => {
```
Comment (kkarimi) [inline frontend/src/helpers.js]: is cate short for category? if so cat is better English

File: frontend/src/Components/Organisation/OrganisationCard.js (hunk 2 above)
Comment (LouiaR) [reply frontend/src/Components/Organisation/OrganisationCard.js]: I think you should check with @DwinaTech

File: frontend/src/helpers.js (hunk 3 above)
Comment (LouiaR) [reply frontend/src/helpers.js]: Done

File: frontend/src/Components/LandingPage/index.js (hunk 1 above)
Comment (LouiaR) [reply frontend/src/Components/LandingPage/index.js]: Done

**Pull request merged**